### DISCUSSION TITLE
Swiftlink emulation?

### DISCUSSION THREAD

**Discussion by xlar54** [frntc/Sidekick64#40] [Ideas]
Hey Frenetic - was just curious about the feasibility of Swiftlink emulation using the core of what youve built here.  Thank you!

**Comment by hpingel** [comment]
Swiftlink emulation  is part of the features of my attempt to create a network enabled Sidekick64 variant:
https://github.com/hpingel/Sidekick64/blob/net-rebase-on-v0.5/README_network.md
